invenio_db/core.py

```diff
@@ -59,6 +61,9 @@ def init_db(self, app, entrypoint_name='invenio_db.models', **kwargs):
             'sqlite:///' + os.path.join(app.instance_path, app.name + '.db')
         )
         app.config.setdefault('SQLALCHEMY_ECHO', app.debug)
+        app.config.setdefault('DB_VERSIONING_USER_MODEL', None)
```

Comment:
@lnielsen do you have any better idea then https://github.com/inveniosoftware/invenio-db/pull/21/files#r47119393 ?